packages/core/src/index.ts

```diff
@@ -3,12 +3,14 @@ import DocumentCache from './common/document-cache.js';
 import TransformManager from './common/transform-manager.js';
 import GlintLanguageServer from './language-server/glint-language-server.js';
 import * as utils from './language-server/util/index.js';
+import * as ts from 'typescript';
```

Comment:
Can we either:
 - make this a type-only import to avoid accidentally pulling in TS at runtime with some seemingly-innocuous change later?
 - just let consumers access `languageServer.service` now that it's non-`private` and leave `ProjectAnalysis` as-is?

Reply:
Good call, I just removed service from `ProjectAnalysis` and dropped the ts import.